core/src/main/java/io/questdb/mp/Worker.java

```diff
@@ -137,7 +137,7 @@ public void run() {
                     }
 
                     if (uselessCounter > sleepThreshold) {
-                        Os.sleep(sleepMs);
+                        Os.sleep(sleepMs * 10);
```

Comment:
is this a test leftover ?

Reply:
Yes, it is. Good spot. Fixed in bbec37baf832db9a846d04e59ab124f8bb84eadc